src/test/scala/nodes/learning/ZCAWhiteningSuite.scala

```diff
@@ -8,18 +8,30 @@ import pipelines._
 
 class ZCAWhiteningSuite extends FunSuite with LocalSparkContext with Logging {
 
-  test("whitening") {
-    val eps = 0.1
-    val nrows = 10000
-    val ndim = 10
+  val nrows = 10000
+  val ndim = 10
 
-    val x = DenseMatrix.rand[Double](nrows, ndim, Gaussian(0.0, 1.0))
+  val x = DenseMatrix.rand[Double](nrows, ndim, Gaussian(0.0, 1.0))
+
+  def fitAndCompare(x: DenseMatrix[Double], eps: Double, thresh: Double): Boolean = {
     val whitener = new ZCAWhitenerEstimator(eps).fitSingle(x)
 
     val wx = whitener(x)
 
-    //Checks max(max(abs(cov(whiten(x))) - eye(10)) < 2*eps
-    assert(max(abs(cov(convert(wx, Double))) - DenseMatrix.eye[Double](ndim)) < 2*eps,
+    //Checks max(max(abs(cov(whiten(x))) - eye(10)) < sqrt(eps)
+    max(abs(cov(convert(wx, Double))) - DenseMatrix.eye[Double](ndim)) < thresh
```

Comment:
So I probably should have commented this earlier, but: the abs should be surrounding the subtraction, not the covariance. Otherwise if covariance is e.g. 0.1 along the diagonals this would still pass even though it shouldn't.